LICENSE.txt

```diff
@@ -1,9 +1,9 @@
-Copyright (c) 2018, Oracle and/or its affiliates. All rights reserved.
+Copyright (c) 2020, Oracle and/or its affiliates. All rights reserved.
```

Comment:
maybe change to 2021?

Reply:
Correct. I will update Copyright notice to the current valid form which is:
`// Copyright (c) 2018, 2020, Oracle and/or its affiliates.`

This form has been validated internally.